test/native/jest_gb.config.js

```diff
@@ -0,0 +1,9 @@
+/** @flow
+ * @format */
+
+const main = require( './jest.config.js' );
+
+module.exports = {
+	...main,
+	moduleDirectories: [ 'node_modules', 'symlinked-packages-in-parent' ],
```

Comment:
I believe we can safely remove `'symlinked-packages-in-parent'` here

Reply:
On a second thought:
Looks like the whole `jest_gb.config.js` file is not really needed?

Reply:
Indeed :D